apps/design/backend/src/ballots.ts

```diff
@@ -73,6 +73,7 @@ export function createBallotPropsForTemplate(
       return baseBallotProps;
 
     default:
+      /* istanbul ignore next - @preserve */
```

Comment:
This won't be honored unless it's within a block statement, so do `default: { … }`.